api/app/jobs/noaa.py

```diff
@@ -0,0 +1,241 @@
+""" A script that downloads weather models from NCEI NOAA HTTPS data server
```

Comment:
Is our naming scheme for python modules/functions/classes based on the source then the model?

e.g here our module is `api/app/jobs/noaa.py`, NOAA being the data source, and we're getting the `GFS` model from this source, so we have the `GFS` class defined here.

Reply:
lol I'm not sure what naming scheme we use in our project overall. We just had a script called `env_canada.py` for fetching the EC models, and I just followed along with that. Open to other suggestions though!